index.html

```diff
@@ -10,28 +10,109 @@
+      </article>
+
+      <article class="project">
+         <img src="resources/P2-IMG.png" alt="imagen projecto 1">
+         <div class="text-content">
+            <div class="pr-description">
+               <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                  sed do eiusmod tempor incididunt ut labore et dolore magna 
+                  aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                   ullamco laboris nisi ut aliquip ea eiusmod.</p>   
+            </div>
+            <ul class="buttons-flex">
+               <li class="li-button">html</li>
+               <li class="li-button">css</li>
+               <li class="li-button">javaScript</li>
+            </ul>
+            <button class="project-button">See this Project -&gt;</button>
+         </div>            
+      </article>
+ 
+      <article class="project">
+         <img src="resources/P3-IMG.png" alt="imagen projecto 1">
+         <div class="text-content">
+            <div class="pr-description">
+               <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                  sed do eiusmod tempor incididunt ut labore et dolore magna 
+                  aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                   ullamco laboris nisi ut aliquip ea eiusmod.</p>   
+            </div>
+            <ul class="buttons-flex">
+               <li class="li-button">html</li>
+               <li class="li-button">css</li>
+               <li class="li-button">javaScript</li>
+            </ul>
+            <button class="project-button">See this Project -&gt;</button>
+         </div>            
+      </article>
+      <article class="project">
+         <img src="resources/P4-IMG.png" alt="imagen projecto 1">
+         <div class="text-content">
+            <div class="pr-description">
+               <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                  sed do eiusmod tempor incididunt ut labore et dolore magna 
+                  aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                   ullamco laboris nisi ut aliquip ea eiusmod.</p>   
+            </div>
+            <ul class="buttons-flex">
+               <li class="li-button">html</li>
+               <li class="li-button">css</li>
+               <li class="li-button">javaScript</li>
+            </ul>
+            <button class="project-button">See this Project -&gt;</button>
+         </div>            
+      </article>
+   </section>
+   <section class="about-me">
+      <h1 class="title">About<br>Me</h1>
+      <p class="parrafo"> 
+         Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+         sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+         Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+         nisi ut aliquip ea eiusmod
+      </p>
+      <button class="about-button">Get my CV -&gt; </button>
+   </section>   
```

Comment:
- [ ] Work section and About me section is incomplete, in the Figma template in the Work section there are six projects, and Languages, Frameworks, and skills are part of the About me section. Kindly complete those sections. ☝🏽 

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).